version/version.go

```diff
@@ -27,7 +29,7 @@ func PrintVersion(c *cli.Context) {
 
 	out.WriteString(fmt.Sprintf("%s %s", c.App.Name, c.App.Version))
 
-	if c.Bool("disable-version-check") {
+	if c.Bool("disable-version-check") || os.Getenv("SOPS_DISABLE_VERSION_CHECK") != "" {
```

Comment:
Why not add the environment variable to https://github.com/getsops/sops/blob/7cfbfd951918a3ff21dda2d5e4fe22c82e4a8dd7/cmd/sops/main.go#L1519-L1522 (with `EnvVar: "SOPS_DISABLE_VERSION_CHECK"`)?

The behavior will be slightly different then from your current implementation (it uses https://pkg.go.dev/strconv#ParseBool to parse the string if it's not empty), but it would avoid using `os.Getenv` explicitly.